.controlplane/Dockerfile

```diff
@@ -53,6 +54,8 @@ RUN apt-get update -qq && \
 COPY --from=build /usr/local/bundle /usr/local/bundle
 COPY --from=build /app /app
 
+RUN chmod +x /app/.controlplane/*.sh
```

Comment:
@borela is this typical?

Reply:
This means the files were committed to the repo without +x permission, you can fix that, commit the files again, and remove that chmod.

Reply:
I did and it didn't fix the issue.